pkg/operation/botanist/constraints_check.go

```diff
@@ -150,22 +138,51 @@ func (b *Botanist) CheckForProblematicWebhooks(ctx context.Context) (gardencorev
 		nil
 }
 
+func buildProblematicWebhookMessage(
+	kind string,
+	configName string,
+	webhookName string,
+	failurePolicy *admissionregistrationv1beta1.FailurePolicyType,
+	timeoutSeconds *int32,
+) string {
+	failurePolicyString := "nil"
+	if failurePolicy != nil {
+		failurePolicyString = string(*failurePolicy)
+	}
+	timeoutString := "and unset timeoutSeconds "
+	if timeoutSeconds != nil {
+		timeoutString = fmt.Sprintf("and %ds timeout ", *timeoutSeconds)
+	}
+	return fmt.Sprintf("%s %q is problematic: webhook %q with failurePolicy %q %smight prevent worker nodes from properly joining the shoot cluster",
```

Comment:
```suggestion
	return fmt.Sprintf("%s %q is problematic: webhook %q with failurePolicy %q %s might prevent worker nodes from properly joining the shoot cluster",
```

Nit, as it's easier to read.

Reply:
Ah sure, it was a leftover.